### ISSUE TITLE
Stats: Add two post stats cards to the Insights page

### ISSUE THREAD

**Issue by jsnmoon** [Automattic/wp-calypso#70854]
# Development

### Must have:

- [x] Add a "Latest Post" card to the Insights page.
( https://github.com/Automattic/wp-calypso/pull/73116 )
- [x] Add a "Most Popular Post" card to the Insights page.
( https://github.com/Automattic/wp-calypso/pull/73116 )
- [x] Remove `Latest post summary` from the bottom of the page.
( https://github.com/Automattic/wp-calypso/pull/73116 )
- [ ] When the latest post is also the most popular, just display the most popular at full width
( _Since the most popular post is hidden for now, we could defer this check until we determine the mechanism to fetch the data of the most popular post._ )
- [x] If the latest post has no stats, hide it and avoid showing only 0s.
( https://github.com/Automattic/wp-calypso/pull/73116 )
- [x] Add a link to the title redirecting to the post stats details page.
( https://github.com/Automattic/wp-calypso/pull/73116 )

### Nice to have:
- [ ] Add mobile support using the dotPager component.
( _Since the most popular post is hidden for now, we could defer this mobile support until we determine the mechanism to fetch the data of the most popular post._ )
- [x] Show a button to add a featured image (on large screens only) if the image is missing ([details](https://github.com/Automattic/wp-calypso/issues/70852#issuecomment-1379285172)). Hide the button on mobile viewports.
( https://github.com/Automattic/wp-calypso/pull/73290 )
- [x] When both cards are shown, truncate the post title to 40 chars.
( https://github.com/Automattic/wp-calypso/pull/73116 )

Must-have items must be completed before closing this issue. Please spin off any unfinished tasks into separate issues.

# Design

If a site has posts, we want to build a visually appealing area that displays the site's latest post (this is any latest post, not tied to any specific author) and the most popular post (again, not tied to any specific author).

![image](https://user-images.githubusercontent.com/4044428/205762580-ed83a4ba-c610-4308-a438-12c76aae1cc3.png)

**Notes**

- "Latest Post" card replaces the current "Latest post summary" card

|Before|After|
|-|-|
|![Screen Shot 2022-12-01 at 5 23 09 PM](https://user-images.githubusercontent.com/214813/205171831-8ccd209f-6fc2-45fc-94f8-634a5b5ddf86.png)|<img width="685" alt="image" src="https://user-images.githubusercontent.com/4044428/205762427-e3c2764d-64f5-4fda-b21b-21e03a2edde7.png">|

- If the latest post IS ALSO the most popular, only one card should display at full width and take the "most popular" headline.
- Mobile version should include a dotPager component

|Before|After|
|-|-|
|![Image](https://user-images.githubusercontent.com/214813/205151752-e828e744-a2d9-4020-8a8e-e1c24253c4ef.png)|![Image](https://user-images.githubusercontent.com/214813/205151539-ddd7e57b-45fc-4c99-ab37-ab95fc4a5c37.png)|

**Specs:**

![Image](https://user-images.githubusercontent.com/214813/205165791-fa1984ff-3d8d-4e93-8a75-00a48e81ade1.png)
![Image](https://user-images.githubusercontent.com/214813/205172786-9a56502d-140a-4580-a8a8-8a592ca8ecfd.png)

**Comment by jsnmoon** [comment]
@dognose24, are there any remaining tasks except for the "Most popular post" component?

If not, let's go ahead and close this task. Could you create a new issue for the "Most popular post" component? I want to add it to our project backlog and remove it from our to-do list for the maintenance sprint.

**Comment by dognose24** [comment]
The new issue for the "Most popular post" component is https://github.com/Automattic/wp-calypso/issues/74008.
